docs/current/data-model-howto.md

```diff
@@ -168,47 +167,107 @@ Boots exist as individual documents under the `boot` subdirectory. They are refe
     └── etcd-proxy.yml
 ```
 
+### Workflows
+
+Workflows exist as individual documents under the `workflows` subdirectory. They are referenced by a self-assigned ID described in the document:
+
+```
+.
+└── workflows
+    ├── clevos-accessor.yml
+    ├── k8s-master.yml
+    └── k8s-node.yml
+```
 
 ### Sites
 
-Sites are represented by individual subdirectories. One directory == one site == one managed group of machines. Each SoT can contain multiple sites. Each of these sites shares the same assets/boot/os configuration files.
+Sites are represented by individual subdirectories under the top level `sites` directory. Each `site` under the `sites` directory corresponds to a managed group
```

Comment:
I'm confused why you mention site directories here but remove those references above. 
https://github.com/CiscoCloud/vaquero-docs/pull/120/files#diff-576b7f479533ca6546369bbc247bc575L68
https://github.com/CiscoCloud/vaquero-docs/pull/120/files#diff-576b7f479533ca6546369bbc247bc575L76

Reply:
That's because site is not a configuration. The idea we are trying to stress out here is that config and the site does not need to coexist.

Reply:
They don't need to coexist, but the sites directory has to be there and sites do contain agent, host, subnet, etc configuration. I think that's even more confusing to say it like that.